### PR TITLE
Features/filters

### PR DESCRIPTION
**Pull request by GuptaEesh**
- added date wise and trending filters
- added suggested user profile column
- added follow/unfollow.

>Follow/unfollow and suggested users

social-follow

>Filters

social-filters

### CODE REVIEW

File: src/backend/db/users.js [hunk 1]
```diff
@@ -8,11 +8,50 @@ import { formatDate } from "../utils/authUtils";
...
+    password: "45!Qqqaaq",
+    bio: "I need to do a lot!",
+    createdAt: formatDate(),
+    updatedAt: formatDate(),
+    avatar:
+      "https://image.freepik.com/free-photo/indoor-picture-cheerful-handsome-young-man-having-folded-hands-looking-directly-smiling-sincerely-wearing-casual-clothes_176532-10257.jpg",
+    following: [],
+    followers: [],
+    bookmarks: [],
   },
```
Comment (ZuberDunge) [inline src/backend/db/users.js:56]: suggestion: add one more profile (mine)

**Review by ZuberDunge — APPROVED** [review]
Well structured code, everything looks fine, good to go from my side.

File: src/backend/db/users.js (hunk 1 above)
Comment (GuptaEesh) [reply src/backend/db/users.js:56]: Yeah sure

**Review by lannisterrr — APPROVED** [review]
Nice and structured Code .
Good use of react redux and slice.
Code seems good to merge.

**Pull request merged**